float64 gps_tow
uint16  gps_week_number
float32 heading_rad
float32 heading_1sigma_rad
uint16  heading_type
bool    use_time
uint16 HEADING_TYPE_TRUE      = 1  
uint16 HEADING_TYPE_MAGNETIC  = 2  
---
bool success
